homeassistant/components/google_tasks/__init__.py

```diff
@@ -4,19 +4,24 @@
 
 from aiohttp import ClientError, ClientResponseError
 
-from homeassistant.config_entries import ConfigEntry
 from homeassistant.const import Platform
 from homeassistant.core import HomeAssistant
 from homeassistant.exceptions import ConfigEntryAuthFailed, ConfigEntryNotReady
 from homeassistant.helpers import config_entry_oauth2_flow
 
 from . import api
 from .const import DOMAIN
+from .exceptions import GoogleTasksApiError
+from .types import GoogleTasksConfigEntry, GoogleTasksData
+
+__all__ = [
+    "DOMAIN",
+]
```

Comment:
I also noticed this was in Google Photos, why do we have that here?

Reply:
My impression was that there is a pattern where integrations are supported to make DOMAIN available in init.  (There were a bunch of refractors to components I have to do this at some point that were merged. I don't remember who pushed on this.)  It's no longer imported for use in code directly, so this is here to keep exporting it.

Perhaps this needs to be a quality scale thing or not a pattern we care about anymore... I don't personally care about it.

Reply:
Like I vaguely remember about it, but I we never really enforced it